src/movement/control_system.cpp

```diff
@@ -29,6 +29,12 @@ namespace robosub
          * Calculate the change in time between each call.
          * TODO: Replace dt with actual message timestamps.
          */
+        if (rate <= 0)
+        {
+            ROS_FATAL("Control system rate is specified as zero or less.");
```

Comment:
Just a note that this could also be a result of the parameter not being specified as well.